### PR TITLE
Re former

### PR DESCRIPTION
**Pull request by Leon-Mbegera**
This project showcases ways of building a form using both the old fashion way and the new way.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to @user
```
Comment (umararshadbut) [inline app/controllers/users_controller.rb]: `redirect_to edit_user_path(<USER_ID`>) if the user was saved (remember to change <USER_ID> to the code that works for the student implementation)

**Review by umararshadbut — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Leon-Mbegera 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes  👇.
- [x] When we edit and submit, the edit is working. But it is not redirecting properly. I'm commenting that where is issue. Please fix it.


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**Umair**

**Comment by Leon-Mbegera** [comment]
I've addressed all required changes:
- redirecting the user update to edit page
- Created a show view

**Review by umararshadbut — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Leon-Mbegera ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
HAPPY CODING :computer:
**Umair**

**Pull request merged**